### PR TITLE
Integration update

### PR DESCRIPTION
**Pull request by mmartin4972**

### CODE REVIEW

**Review by cgiger00 — CHANGES_REQUESTED** [review]

File: base_station/gui/src/components/RATask.vue [hunk 1]
```diff
@@ -311,15 +351,25 @@ export default {
     grid-area: map;
   }
 
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (CameronTressler) [reply base_station/gui/src/components/RATask.vue]: Uh whoops, there goes my faith in vscode. Sorry about this

[the remark this answers is not in the document]
Comment (CameronTressler) [reply base_station/gui/src/components/RATask.vue]: wait these aren't in the code I pushed, they're all outdated. Maybe Github didn't update and you made these comments right after I pushed or something?... Idk but these should be fixed

**Review by CameronTressler — APPROVED** [review]

**Comment by zwgold** [comment]
Looks fine to me. There are 2 recent commits to main that are not merged in, do we want to include those as well with this PR?

**Review by zwgold — APPROVED** [review]

File: base_station/gui/src/components/RATask.vue (hunk 1 above)
[the remark this answers is not in the document]
Comment (cgiger00) [reply base_station/gui/src/components/RATask.vue]: no i think these are from the merge/rebase matt is doing

**Review by cgiger00 — COMMENTED** [review]
can this be rebased again? main has been updated w some more significant changes (like the drive and sim stuff) recently

**Review by cgiger00 — APPROVED** [review]
lgtm

**Pull request merged**